src/command/defaults/TimeCommand.php

```diff
@@ -72,16 +72,19 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 			return true;
 		}elseif($args[0] === "query"){
 			if(!$sender->hasPermission("pocketmine.command.time.query")){
-				$sender->sendMessage($sender->getServer()->getLanguage()->translateString(TextFormat::RED . "%commands.generic.permission"));
+				$sender->sendMessage(TextFormat::RED . "%commands.generic.permission");
 
 				return true;
 			}
+
```

Comment:
This formatting change can be reverted

Reply:
To be exact, please do not include irrelevant formatting changes at your discrete so as not to corrupt the commit diff.